extensions/sql-database-projects/src/dialogs/utils.ts

```diff
@@ -30,34 +31,59 @@ export function getAgreementDisplayText(agreementInfo: AgreementInfo): string {
 	return constants.eulaAgreementText(agreementInfo.link!.text);
 }
 
-export function getDockerBaseImages(): DockerImageInfo[] {
-	return [
-		{
-			name: `${constants.sqlServerDockerRegistry}/${constants.sqlServerDockerRepository}:2017-latest`,
-			agreementInfo: {
-				link: {
-					text: constants.eulaAgreementTitle,
-					url: constants.sqlServerEulaLink,
-				}
-			}
-		},
-		{
+export function getDockerBaseImages(target: string): DockerImageInfo[] {
+	if (target === constants.targetPlatformToVersion.get(SqlTargetPlatform.sqlAzure)) {
+		return [{
 			name: `${constants.sqlServerDockerRegistry}/${constants.sqlServerDockerRepository}:2019-latest`,
+			displayName: constants.sqlServer2019,
```

Comment:
sql server 2019 is an option if the target platform is Azure?

Reply:
yes for now, we don't have image container for Azure yet